ArchiSteamFarm/IPC/ArchiKestrel.cs

```diff
@@ -34,6 +34,8 @@ namespace ArchiSteamFarm.IPC {
 	internal static class ArchiKestrel {
 		private const string ConfigurationFile = nameof(IPC) + ".config";
 
+		internal static string WebsiteDirectory { get; private set; } = Path.Combine(SharedInfo.HomeDirectory, SharedInfo.WebsiteDirectory);
```

Comment:
And like I said above, you don't need absolute directory here because `HomeDirectory` is already default for kestrel, so just use relative.